commit 7f808daeb347b1b40c39e7c83e0eec2389f29403 (refs/changes/03/2703/1)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2017-10-03T18:05:11+00:00 (2 years ago)
    
    backout: "Add useful contrib/ modules.  Surprise!, these are useful in production!!!" (needs more work)

diff --git a/Makefile.postgres b/Makefile.postgres
index c02e935..cdd2ecd 100644
--- a/Makefile.postgres
+++ b/Makefile.postgres
@@ -82,36 +82,9 @@ pg96:
 	cd $(DEPSDIR)/postgresql96 && env \
 		CFLAGS=-m64 LDFLAGS=-m64 \
 		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
-	cd $(DEPSDIR)/postgresql96/contrib/auto_explain && env \
-		CFLAGS=-m64 LDFLAGS=-m64 \
-		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
-	cd $(DEPSDIR)/postgresql96/contrib/oid2name && env \
-		CFLAGS=-m64 LDFLAGS=-m64 \
-		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
-	cd $(DEPSDIR)/postgresql96/contrib/pageinspect && env \
-		CFLAGS=-m64 LDFLAGS=-m64 \
-		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
-	cd $(DEPSDIR)/postgresql96/contrib/pg_buffercache && env \
-		CFLAGS=-m64 LDFLAGS=-m64 \
-		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
-	cd $(DEPSDIR)/postgresql96/contrib/pg_freespacemap && env \
-		CFLAGS=-m64 LDFLAGS=-m64 \
-		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
-	cd $(DEPSDIR)/postgresql96/contrib/pg_prewarm && env \
-		CFLAGS=-m64 LDFLAGS=-m64 \
-		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
 	cd $(DEPSDIR)/postgresql96/contrib/pg_stat_statements && env \
 		CFLAGS=-m64 LDFLAGS=-m64 \
 		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
-	cd $(DEPSDIR)/postgresql96/contrib/pg_visibility && env \
-		CFLAGS=-m64 LDFLAGS=-m64 \
-		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
-	cd $(DEPSDIR)/postgresql96/contrib/pgstattuple && env \
-		CFLAGS=-m64 LDFLAGS=-m64 \
-		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
-	cd $(DEPSDIR)/postgresql96/contrib/vacuumlo && env \
-		CFLAGS=-m64 LDFLAGS=-m64 \
-		$(MAKE) install DESTDIR="$(RELSTAGEDIR)/root"
 	cd $(DEPSDIR)/pg_repack && env \
 		CFLAGS=-m64 \
 		PATH=$(RELSTAGEDIR)/root/$(PG96DIR)/bin:$$PATH \
